internal/langserver/handlers/service.go

```diff
@@ -272,6 +272,17 @@ func (svc *service) Assigner() (jrpc2.Assigner, error) {
 
 			return handle(ctx, req, svc.TextDocumentFormatting)
 		},
+		"textDocument/signatureHelp": func(ctx context.Context, req *jrpc2.Request) (interface{}, error) {
+			err := session.CheckInitializationIsConfirmed()
+			if err != nil {
+				return nil, err
+			}
+
+			ctx = ilsp.WithClientCapabilities(ctx, cc)
+			ctx = ilsp.ContextWithClientName(ctx, &clientName)
```

Comment:
What do we need the client name for here? 🤔

Reply:
Good catch, copy/waste from `hover`